src/main/kotlin/com/example/template/Application.kt

```diff
@@ -1,10 +1,18 @@
 package com.example.template
 
+import org.springframework.boot.CommandLineRunner
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.context.annotation.Bean
 
 @SpringBootApplication
-class Application
+class Application {
+    // Print out the Apollo Studio Sandbox URL for querying this application
+    @Bean
+    fun commandLineRunner() = CommandLineRunner {
+        println("Apollo Studio Sandbox URL: https://studio.apollographql.com/sandbox/explorer?endpoint=http://localhost:8080/graphql")
```

Comment:
this is odd way of doing this -> guess it works but why not embed it in a local file? https://www.apollographql.com/docs/graphos/explorer/embed-explorer/

Reply:
Easier for consistency across templates to not have to figure out how to serve a web page (which many people will just remove for their deployment anyway).